### DISCUSSION TITLE
[source-google-drive] support excel format

### DISCUSSION THREAD

**Discussion by KevinConseilxyz** [airbytehq/airbyte#33423] [Connector Ideas and Features]
Hello :)

**Feature Request:** Add excel format as supported format of the source google drive connector

**Use Cases:** 

- Weekly sales in excel files are added every week in a google drive folder and need to be loaded to Snowflake weekly
- Daily user log snapshot in excel file are added in a google drive folder and need to be loaded to Snowflake daily

Happy to give further information :)

Kevin

**Comment by marcosmarxm** [comment]
Thanks for the request @kevinshareGmbH !

[a comment by an automated account is not shown]
